Arashayo/TabletopHaven
# Add chat

## Related Issue
https://app.clickup.com/t/86bxnh5b0

## Changes list
Added chat

## Testing instructions
run `node install-with-users.js 200`
run `node run.js`
create some groups, try sending messages, log on different browser with second account, check how sending messages works


## Screenshots (if appropriate):
